### PR TITLE
Attractor Size is fixed

### PR DESCRIPTION
**Pull request by shreyasbapat**
See this: 
screenshot from 2018-06-12 00-36-50
Solves #385

### CODE REVIEW

File: src/poliastro/plotting.py [hunk 1]
```diff
@@ -160,12 +160,14 @@ def _project(self, rr):
     def _redraw_attractor(self, min_radius=0 * u.km):
         radius = max(self._attractor.R.to(u.km).value, min_radius.to(u.km).value)
         color = BODY_COLORS.get(self._attractor.name, "#999999")
-        self._attractor_radius = radius
 
         for attractor in self.ax.findobj(match=mpl.patches.Circle):
             attractor.remove()
 
-        self.ax.add_patch(mpl.patches.Circle((0, 0), radius, lw=0, color=color))
+        if radius < self._attractor_radius:
```
Comment (Juanlu001) [inline src/poliastro/plotting.py]: Shouldn't this have a unit, as it's done in the other two cases (`np.inf * u.km`)? I'm just asking, if the code works like this then it's fine for me.

Comment (shreyasbapat) [reply src/poliastro/plotting.py]: Although it was working, but then it seems more uniform. Thanks for the suggestion.

**Comment by Juanlu001** [comment]
Interesting:

```
>>> 1 < np.inf
True
>>> 1 * u.km < np.inf
True
>>> 1 * u.km < np.inf * u.km
True
>>> 1 < np.inf * u.km
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/juanlu/.miniconda36/envs/py36/lib/python3.6/site-packages/astropy/units/quantity.py", line 620, in __array_ufunc__
    converters, unit = converters_and_unit(function, method, *inputs)
  File "/home/juanlu/.miniconda36/envs/py36/lib/python3.6/site-packages/astropy/units/quantity_helper.py", line 556, in converters_and_unit
    .format(function.__name__))
astropy.units.core.UnitsError: Can only apply 'greater' function to dimensionless quantities when other argument is not a quantity (unless the latter is all zero/infinity/nan)
```

**Comment by Juanlu001** [comment]
Thanks @shreyasbapat!

**Comment by shreyasbapat** [comment]
Indeed! Why is this happening? Any clue?

**Comment by Juanlu001** [comment]
I would expect `1 * u.km < np.inf` to fail. The thing is that everything is less than `np.inf`, so I'm not sure how would I fix this. In any case, this is an `astropy.units` issue, feel free to report if you want :)

**Pull request merged**